### ISSUE TITLE
Headings

### ISSUE THREAD

**Issue by yvonnetse** [BraftonSupport/braftonium#70]
Headings do not skip levels when descending. H1 cannot be followed by H3, etc.

**Comment by yvonnetse** [comment]
Not much I can do, it depends on the copy of the page.
